lib/record_store/provider.rb

```diff
@@ -19,6 +19,8 @@ def provider_for(zone_name)
           'DynECT'
         when /googledomains\.com\z/
           'GoogleCloudDNS'
+        when /ns1\.com\z/
```

Comment:
Not a blocker at all, and I know this is done the same way above, but wouldn't we want to have the a dot included at the beginning of the matching here? Any domain that would be `anotherns1.com` would match `ns1` too.

Reply:
This is a good point.  Wanna handle this in a follow-on PR, @gooallen?

Reply:
@cttttt Will look into it.